src/test/java/eu/yals/test/ui/VaadinTest.java

```diff
@@ -30,155 +30,160 @@
+        }
+        if (testBrowsers.contains(TestApp.Browser.IE)) {
+            browsers.add(BrowserUtil.ie11());
+        }
+        if (testBrowsers.contains(TestApp.Browser.EDGE)) {
+            browsers.add(BrowserUtil.edge());
+        }
+
+        return browsers;
+    }
 
-  @SuppressWarnings("rawtypes")
-  private void addTestNameToDriver() throws IllegalAccessException, ClassCastException {
-    Class parallelTest = getClass();
-    do {
-      if (parallelTest == null) break;
-      parallelTest = parallelTest.getSuperclass();
-    } while (!parallelTest.getSimpleName().equals("ParallelTest"));
+    @SuppressWarnings("rawtypes")
+    private void addTestNameToDriver() throws IllegalAccessException, ClassCastException {
+        Class parallelTest = getClass();
+        do {
+            if (parallelTest == null) break;
+            parallelTest = parallelTest.getSuperclass();
+        } while (!parallelTest.getSimpleName().equals("ParallelTest"));
 
-    if (parallelTest == null) return;
+        if (parallelTest == null) return;
 
-    Field[] parallelTestFields = parallelTest.getDeclaredFields();
+        Field[] parallelTestFields = parallelTest.getDeclaredFields();
 
-    Field driverConfigurationField = null;
-    for (Field f : parallelTestFields) {
-      if (f.getName().equals("driverConfiguration")) {
-        driverConfigurationField = f;
-      }
+        Field driverConfigurationField = null;
+        for (Field f : parallelTestFields) {
+            if (f.getName().equals("driverConfiguration")) {
+                driverConfigurationField = f;
+            }
+        }
+        if (driverConfigurationField == null) return;
+        driverConfigurationField.setAccessible(true);
+        SetupDriver sd = (SetupDriver) driverConfigurationField.get(this);
+        if (sd == null || sd.getDesiredCapabilities() == null) return;
+        sd.getDesiredCapabilities().setCapability("name", testName);
+        sd.getDesiredCapabilities().setCapability("build", BUILD_NAME);
     }
-    if (driverConfigurationField == null) return;
-    driverConfigurationField.setAccessible(true);
-    SetupDriver sd = (SetupDriver) driverConfigurationField.get(this);
-    if (sd == null || sd.getDesiredCapabilities() == null) return;
-    sd.getDesiredCapabilities().setCapability("name", testName);
-    sd.getDesiredCapabilities().setCapability("build", BUILD_NAME);
-  }
-
-  private String setTestNameFromTestDescription(Description description) {
-    String rawMethodName = description.getMethodName();
-    String[] methodAndBrowserInfo = rawMethodName.split("\\[");
-    if (methodAndBrowserInfo.length > 0) {
-      String method = methodAndBrowserInfo[0];
-      return String.format("%s.%s", description.getTestClass().getName(), method);
-    } else {
-      return String.format("%s.%s", description.getTestClass().getName(), rawMethodName);
+
+    private String setTestNameFromTestDescription(Description description) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter description should be final.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)